workshops/templates/workshops/profileupdaterequest.html

```diff
@@ -118,6 +109,16 @@
         {% endif %}
         </td>
     </tr>
+    <tr><th colspan="3" class="text-center">Additional information (not being saved to the database)</th></tr>
```

Comment:
Gotcha.  We should store ORCID and occupation eventually.  Other than 
that, happy to merge this.